scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -846,6 +846,21 @@ cdef class RigidTransformation:
         This implements the exponential map that converts 6-dimensional real
         vectors to SE(3).
 
+        An exponential coordinate vector consists of 6 elements: the first 3 encode
```

Comment:
Would be nice to explicitly write out the whole vector `(rx, ry, rz, tx, ty, tz)`.